rclrs/src/rcl_wrapper.h

```diff
@@ -5,3 +5,5 @@
 #include <rmw/types.h>
 #include <rosidl_typesupport_introspection_c/field_types.h>
 #include <rosidl_typesupport_introspection_c/message_introspection.h>
+
+const size_t rmw_gid_storage_size = RMW_GID_STORAGE_SIZE;
```

Comment:
According to https://github.com/rust-lang/rfcs/blob/master/text/0430-finalizing-naming-conventions.md#general-naming-conventions, constants in Rust should be uppercase, do you know if there's a way to tell `bindgen` to tweak any of the variables in a header file so that it's transformed to uppercase?